CHANGELOG.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Changelog
+menuTitle: Changelog
+category: Guides
+position: 103
+---
+
+Since 1.17.1, chproxy follows [semantic versioning](https://semver.org/).
```

Comment:
Small remark: currently all versions have prefix. Therefore instead of `1.17.1` we have `v1.17.1`